tests/dialects/test_builtin.py

```diff
@@ -58,6 +60,51 @@ def test_IntegerType_size():
     assert IntegerType(64).size == 8
 
 
+def test_IntegerType_normalize():
+    s8 = IntegerType(8, Signedness.SIGNED)
```

Comment:
Extreme nit but I believe `si` is usually used in mlir as a prefix for signed integers

Reply:
oh this is great, I couldn't remember it, and it's not in the IntegerAttr doc, of course